lib/metadata/bucketfile/backend.js

```diff
@@ -0,0 +1,450 @@
+import net from 'net';
+import fs from 'fs';
+import cluster from 'cluster';
+import events from 'events';
+import assert from 'assert';
+
+import level from 'level';
+import multilevel from 'multilevel';
+import sublevel from 'level-sublevel';
+import arsenal from 'arsenal';
+import { logger } from '../../utilities/logger';
+
+import { errors } from 'arsenal';
```

Comment:
Couldn't this be `const errors = arsenal.errors;` since `arsenal` is already imported once?